install-dev/upgrade/sql/1.7.5.0.sql

```diff
@@ -7,3 +7,17 @@ SET NAMES 'utf8';
 
 ALTER TABLE `PREFIX_cms_lang`
 	ADD `head_seo_title` varchar(255) DEFAULT NULL AFTER `meta_title`;
+
+ALTER TABLE `PREFIX_currency` ADD `numeric_iso_code` varchar(3) NOT NULL DEFAULT '0' AFTER `iso_code`;
+ALTER TABLE `PREFIX_currency` ADD `precision` int(2) NOT NULL DEFAULT 6 AFTER `numeric_iso_code`;
```

Comment:
You added an index in the table, which needs to be added here as well

Reply:
which index are you talking about ?